themes/src/main/themes/VAADIN/themes/valo/components/_treegrid.scss

```diff
@@ -1,49 +1,43 @@
 @import "grid";
 
 /** Expander button visual - expanded */
-$tg-expander-char-expanded: '\f0d7' !default;
+$v-treegrid-expander-char-expanded: '\f0d7' !default;
 
 /** Expander button visual - collapsed */
-$tg-expander-char-collapsed: '\f0da' !default;
-
-/** Expander button width */
-$tg-expander-width: 10px !default;
-
-/** Expander button right side padding */
-$tg-expander-padding: 10px !default;
+$v-treegrid-expander-char-collapsed: '\f0da' !default;
 
 @mixin valo-treegrid ($primary-stylename: v-treegrid) {
 
   @include valo-grid($primary-stylename);
 
   // Expander with and item indentation constants
-  $indent: $tg-expander-width + $tg-expander-padding;
+  $indent:2ex !default;
 
   // Classname for depth styling
   $class-depth: depth !default;
 
   .#{$primary-stylename}-expander {
     display: inline-block;
-    width: $tg-expander-width;
-    padding-right: $tg-expander-padding;
+    width: 1ex;
+    padding-right: 1ex;
 
     &::before {
```

Comment:
Nit: inconsistent use of colon and double colon for pseudo elements (`:before` vs. `::before`). The latter is the correct syntax (so it’s correct here) and works in all the supported browsers.

For reference: http://www.growingwiththeweb.com/2012/08/pseudo-classes-vs-pseudo-elements.html